app/src/main/java/no/usn/mob3000/ui/components/DangerousActionDialogue.kt

```diff
@@ -35,7 +34,7 @@ fun DangerousActionDialogue(
     confirmText: String = stringResource(R.string.component_dialogue_dangerous_action_button_confirm),
     dismissText: String = stringResource(R.string.component_dialogue_dangerous_action_button_dismiss),
     colorContainer: Color = MaterialTheme.colorScheme.surface,
-    colorButtonDismiss: Color = DefaultButton,
+    colorButtonDismiss: Color = MaterialTheme.colorScheme.primary,
     colorButtonConfirm: Color = Color.Red,
```

Comment:
Replacing the `Color.Red` with `MaterialTheme.colorScheme.`'s `error` or `onError` would be a nice touch. The color scheme parameter can simply be set to `Color.Red` for now.